docs/modules/integrate/pages/kafka-connector.adoc

```diff
@@ -75,6 +75,12 @@ committing using `enable.auto.commit` and configuring
 link:https://kafka.apache.org/22/documentation.html[Kafka documentation]
 for the descriptions of these properties.
 
+You can also explicitly specify exact initial offsets for the Kafka source using `TopicsConfig` parameter.
+Note that initial offsets provided in `topicConfig` will always have a priority over offsets stored in Kafka or associated with given consumer group.
+Those offsets are used only when the job is started for the first time after submission.
```

Comment:
technically this is until the first snapshot is taken, but IMO it is too much detail

Reply:
I agree.